src/app/pages/courses/course-card/course-card.component.ts

```diff
@@ -35,4 +41,8 @@ export class CourseCardComponent {
   get creationTime(): Date {
     return this.course.creationTime;
   }
+
+  deleteCard(): void {
```

Comment:
You have full object, probably I would emit the object, it should be more flexible , subscribers will have access to object , not only ID